packages/edit-post/src/editor.js

```diff
@@ -102,6 +106,7 @@ function Editor( {
 					? getEditedPostTemplate()
 					: null,
 			post: postObject,
+			isEditorReady: __unstableIsEditorReady(),
```

Comment:
Reading the coding standards and wondering if this should be returned with the `__unstable` prefix too. 

What am I missing? This is mainly for my own education rather than an opinion either way :)

Reply:
I was following the style established elsewhere in the code where `__unstable` and `__experimental` is often removed, for example here:

https://github.com/WordPress/gutenberg/blob/04c7c760dc480ae2d2eae295ca5bab6c766d31a9/packages/edit-post/src/components/layout/index.js#L17-L25

Also, if I'm not mistaken, it seems that the `__unstable` and `__experimental` prefix is there for people using Gutenberg packages in their projects, so it's important not to document these functions and tell people to use them and so forth. Here it's just used locally and not exported anywhere.

That's just my reasoning for renaming it, but I'll put the `__unstable` prefix back if you think that would be better.